.github/workflows/ci.yml

```diff
@@ -0,0 +1,128 @@
+              run: echo "::set-env name=DATABASE_URL::pgsql://postgres:password@127.0.0.1:${{ job.services.postgres.ports[5432] }}/postgres"
+
+            - name: Install PHP
+              uses: shivammathur/setup-php@v1
+              with:
+                  php-version: ${{ env.PHP_VERSION }}
+                  coverage: none
+                  extensions: ctype, iconv, intl
+                  ini-values: date.timezone=Europe/Paris, memory_limit=-1, session.gc_probability=0, opcache.enable=1, opcache.enable_cli=1, opcache.memory_consumption=256, opcache.max_accelerated_files=20000, opcache.validate_timestamps=0, realpath_cache_size=4096K, realpath_cache_ttl=600
+                  tools: cs2pr
+
+            - name: Install Node.js
+              uses: actions/setup-node@v1
+              with:
+                  node-version: ${{ env.NODE_VERSION }}
+
+            - uses: actions/checkout@v2
+
+            - name: Get Composer cache directory
+              id: composer-cache
+              run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+            - name: Cache Composer dependencies
+              uses: actions/cache@v1
+              with:
+                  path: ${{ steps.composer-cache.outputs.dir }}
+                  key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
+                  restore-keys: ${{ runner.os }}-composer-
+
+            - run: composer install --no-progress --no-suggest --prefer-dist --optimize-autoloader
+
+            - name: Cache node_modules
+              uses: actions/cache@v1
+              with:
+                  path: ${{ github.workspace }}/node_modules
+                  key: ${{ runner.os }}-node-modules-${{ hashFiles('**/yarn.lock') }}
+                  restore-keys: |
+                      ${{ runner.os }}-node-modules-
+
+            - run: yarn install --frozen-lockfile
+
+            - run: yarn encore production
+
+            - name: Bootstrap database
+              run: |
+                  bin/post-install-dev.sh
+
+            - run: composer validate
+              if: always()
+
+            - run: php bin/console lint:twig templates
+              if: always()
+
+            - run: php bin/console lint:yaml config --parse-tags
+              if: always()
+
+            - run: php bin/php-cs-fixer fix --verbose --diff --dry-run --format=checkstyle | cs2pr
+              if: always()
+
+            - run: bin/phpcs --warning-severity=0 --report=checkstyle | cs2pr
+              working-directory: ${{ matrix.working_directory }}
+              if: always()
+
+            - run: php bin/console doctrine:schema:validate
+              if: always()
+
+            - name: Run PHPStan
+              run: php bin/phpstan.phar analyse --configuration phpstan.neon --level max --no-progress . --error-format=checkstyle | cs2pr
+              if: always()
+
+            - run: bin/phpunit --printer mheap\\GithubActionsReporter\\Printer
```

Comment:
Can't we keep the phpdbg code coverage report ?

Reply:
I don't know; how was it generated before ?
Also, are you reffering to https://github.com/krakjoe/phpdbg this ? because it's archived and unmaintained for 6 years 😱

Reply:
Also: the action i'm using to setup PHP can support code coverage using Xdebug or PCOV (which is faster); see : https://github.com/marketplace/actions/setup-php-action#signal_strength-coverage-support

Maybe re-enable code coverage in another PR ?

Reply:
We'll re-enable code coverage later, ok. But you should check what was existing before creating a PR updating everything, don't you think ? ;-) 

https://gitlab.com/mRoca/resop/-/jobs/491684797 > you can see the code coverage report. I've removed the coverage badge from the README only because there is a but on displaying it on github, you can see it there : https://gitlab.com/mRoca/resop/-/tree/2fc11fd3b22834772596da8f36f008ba7f3bf198#contributing

And for your information, the krakjoe/phpdbg has been archived 6 years ago because phpdbg has been added to the php-src repository itself, from php5.6.

Reply:
Sure, you got a point; I'd assume you had no CI since the badge were broken, and even the link you provided lands on 404; My bad :)